src/common/procedure/src/store/state_store.rs

```diff
@@ -189,7 +189,7 @@ impl StateStore for ObjectStateStore {
 
     async fn batch_delete(&self, keys: &[String]) -> Result<()> {
         self.store
-            .remove(keys.to_vec())
+            .delete_iter(keys.to_vec())
```

Comment:
```suggestion
            .delete_iter(keys)
```

Reply:
Addressed in [`20114a5` (#5354)](https://github.com/GreptimeTeam/greptimedb/pull/5354/commits/20114a5320607a41f1c8c84b9a9ed6548fc32064)